modules/file-system/filestore/outputs.tf

```diff
@@ -50,3 +50,13 @@ output "mount_runner" {
   EOT
   value       = local.mount_runner
 }
+
+output "filestore_id" {
```

Comment:
The way I read the the [filestore] resource docs, the `id` attributes should be exported only after creation. This should cause Terraform to order resources correctly.

[filestore]: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/filestore_instance#attributes-reference

Reply:
That is my understanding. Is there any action to be taken with respect to this comment?

Reply:
No, but I'm contrasting it with the capacity output value (where I have a concern) below.